docs/client/full-api/api/accounts.md

```diff
@@ -318,7 +318,7 @@ are called with a single argument, the attempt info object:
 
 <dl class="objdesc">
 {{#dtdd name="type" type="String"}}
-  The service name, such as "password" or "twitter".
+  (Server only) The service name, such as "password" or "twitter".
```

Comment:
Let's be more explicit for now, and I'll clean up the docs later (I got really confused about how this is related to this PR)

Something like:

> The service name, such as "password" or "twitter". For `Accounts.onLogin` and `Accounts.onLoginFailure`, this data is only passed on the server.